src/brevitas/core/scaling/runtime.py

```diff
@@ -81,21 +75,12 @@ def __init__(
             self,
             restrict_scaling_impl: Module,
             restrict_threshold_impl: Module,
-            scaling_shape: Tuple[int, ...],
-            scaling_min_val: Optional[float],
-            affine_rescaling: bool,
```

Comment:
In my mind, the solution would have been to remove these two boolean flag, and apply a similar trick to the one we use for scaling_min_val, where we assume it's None, and if it isn't, then we instantiate the ClampingModule.

In this case, `affine_rescaling_init_val` would become an optional input set to None, and if it is not None, then we could create `_AffineRescaling` class with the init value, otherwise identity as we already do below.

The init value can either be a scalar, that will be automatically broadcasted to scaling_shape, or something else (still compatible with scaling_shape)